lib/src/models/reaction.dart

```diff
@@ -0,0 +1,19 @@
+class Reaction {
+  Reaction({
+    required this.reactions,
+    required this.reactedUserIds,
+  });
+
+  factory Reaction.fromJson(Map<String, dynamic> json) => Reaction(
+        reactions: json['reactions'],
+        reactedUserIds: json['reactedUserIds'],
+      );
+
+  List<String> reactions;
```

Comment:
add `final` keyword.

Reply:
Here we haven't added final keyword because when user add other emoji that time list will be modified that is the main reason for same.